packages/astro/e2e/multiple-frameworks.test.js

```diff
@@ -14,89 +14,89 @@ test.afterAll(async () => {
 });
 
 test.describe('Multiple frameworks', () => {
-	test('React counter', async ({ astro, page }) => {
+	test.skip('React counter', async ({ page }) => {
 		await page.goto('/');
 
-		const counter = await page.locator('#react-counter');
+		const counter = page.locator('#react-counter');
 		await expect(counter, 'component is visible').toBeVisible();
 
-		const count = await counter.locator('pre');
+		const count = counter.locator('pre');
 		await expect(count, 'initial count is 0').toHaveText('0');
 
-		const increment = await counter.locator('.increment');
+		const increment = counter.locator('.increment');
 		await increment.click();
 
 		await expect(count, 'count incremented by 1').toHaveText('1');
 	});
 
-	test('Preact counter', async ({ astro, page }) => {
+	test('Preact counter', async ({ page }) => {
```

Comment:
does this mean that awaiting _anywhere_ in these tests introduces flakiness? In that case, we should probably make these non-async functions and leave a comment somewhere. 

Alternatively, you can force these tests to run serially when you run them. That's essentially what you're forcing by making them all synchronous, but fixing at the test-runner level would prevent future bugs.

Reply:
No, these awaits weren't part of the flakiness. They were just unnecessary. The source of the flakiness was the React test.

Reply:
Roger that, thanks for confirming!